book_inventory.c

```diff
@@ -0,0 +1,47 @@
+#include <stdio.h>
+#include <string.h>
+
+#define LENGTH_TITLE (41)
+#define LENGTH_AUTHOR (31)
+
+char * s_gets(char *state_book, int number_of_book);
+
+struct book_price{
+    char tittle [LENGTH_TITLE];
+    char author [LENGTH_AUTHOR];
+    float value_of_book;
+};
+
+int main(void)
+{
+    struct book_price book;
+    printf("Please enter the book tittle.\n");
+    s_gets(book.tittle, LENGTH_TITLE);
+    printf("Now enter the author\n");
+    s_gets(book.author, LENGTH_AUTHOR);
+    printf("Now enter the value.\n");
+    scanf("%f",&book.value_of_book);
+    printf("%s by %s: $%.2f\n",book.tittle, book.author, book.value_of_book);
+    printf("%s: \"%s\" ($%.2f)\n",book.author, book.tittle, book.value_of_book);
+    printf("Done.\n");
+
+    return 0;
+}
+
+char * s_gets(char *st, int number_of_book)
```

Comment:
```suggestion
char * s_gets(char *st, int length)
```

Since this function is `s_gets` (a generic name), I think this parameter needs to have a different name. The same goes for the variables in this function.